README.md

```diff
@@ -84,28 +84,32 @@ The `sys` publisher is not a real node ID, but it's also not a special case valu
 - UpArrow/DownArrow or CTRL+P/CTRL+N to move up and down through command history
 - CTRL+R to search history, CTRL+R again to toggle through search results, CTRL+G to cancel search
 
-- `/message <address> <json>`: send an inter-process message. <address> is formatted as <node>@<process_id>. <process_id> is formatted as <process_name>:<package_name>:<publisher_node>.
-    - Example: `/message our@net:distro:sys diagnostics`
+- `m <address> <json>`: send an inter-process message. <address> is formatted as <node>@<process_id>. <process_id> is formatted as <process_name>:<package_name>:<publisher_node>.
+    - Example: `m our@net:distro:sys diagnostics`
     - `our` will always be interpolated by the system as your node's name
-    - Can also use `/m` for same command: `/m our@net:distro:sys diagnostics`
-- `/app <address>`: set the terminal to a mode where all messages go to a specific app. To clear this selection, use `/app clear` or simply `/app`. This is useful for apps that have a command line interface.
+    - Can also use `m` for same command: `m our@net:distro:sys diagnostics`
+<!-- - `/app <address>`: set the terminal to a mode where all messages go to a specific app. To clear this selection, use `/app clear` or simply `/app`. This is useful for apps that have a command line interface.
```

Comment:
oh so I removed this because it causes some complication with the interaction between `m` and `terminal`. I could add it back, but I'm not sure it's the right solution. I think the right thing to do later would be to have aliases like `export FOO=our@net:distro:sys` and then calling `m FOO diagnostics`